security/models.py

```diff
@@ -236,8 +239,7 @@ class UIMeta:
 
 class InputLoggedRequest(LoggedRequest):
 
-    user = models.ForeignKey(django_settings.AUTH_USER_MODEL, verbose_name=_('user'), null=True, blank=True,
-                             on_delete=models.SET_NULL)
+    user_id = models.TextField(verbose_name=_('user ID'), null=True, blank=True, db_index=True)
```

Comment:
No user cannot be stored as FK, because of more connections or more databases in the future

Reply:
Do really more connections matter? I understand that for more databases, this is legit, but Foreign key should work with multiple connections to the same database, or not?

Reply:
yes it matter. Objects created in connection is not accessible in the secondary connections